wpk/generate_wpk_package.sh

```diff
@@ -9,55 +9,59 @@
 # License (version 2) as published by the FSF - Free Software
 # Foundation.
 
-CURRENT_PATH="$( cd $(dirname $0) ; pwd -P )"
+CURRENT_PATH="$( cd $(dirname ${0}) ; pwd -P )"
 LINUX_BUILDER="unified_linux_wpk_builder"
 LINUX_BUILDER_DOCKERFILE="${CURRENT_PATH}/unified/linux"
 WIN_BUILDER="windows_wpk_builder"
 WIN_BUILDER_DOCKERFILE="${CURRENT_PATH}/windows"
 CHECKSUM="no"
+lsION_PATH="/var"
```

Comment:
This variable is unused.

Reply:
Fixed https://github.com/wazuh/wazuh-packages/pull/408/commits/266a8eff05ec92416c5eee4eb6f8adbfbf3a07ea